__DEFINES/map.dm

```diff
@@ -0,0 +1,40 @@
+///Map generation defines
+#define BIOME_LOWEST_HUMIDITY "biome_lowest_humidity"
+#define BIOME_LOW_HUMIDITY "biome_low_humidity"
+#define BIOME_MEDIUM_HUMIDITY "biome_medium_humidity"
+#define BIOME_HIGH_HUMIDITY "biome_high_humidity"
+#define BIOME_HIGHEST_HUMIDITY "biome_highest_humidity"
+
+#define BIOME_COLDEST "coldest"
+#define BIOME_COLD "cold"
+#define BIOME_WARM "warm"
+#define BIOME_TEMPERATE "perfect"
+#define BIOME_HOT "hot"
+#define BIOME_HOTTEST "hottest"
+
+#define BIOME_COLDEST_CAVE "coldest_cave"
+#define BIOME_COLD_CAVE "cold_cave"
+#define BIOME_WARM_CAVE "warm_cave"
+#define BIOME_HOT_CAVE "hot_cave"
+
+#define CLIMATE_ARCTIC	/datum/climate/arctic
+#define CLIMATE_TEMPERATE /datum/climate/temperate
+#define CLIMATE_TROPICAL /datum/climate/tropical
+#define CLIMATE_DESERT /datum/climate/desert
+#define CLIMATE_LAVA /datum/climate/lava
+#define CLIMATE_TOXIC /datum/climate/wasteland
+#define CLIMATE_XENO /datum/climate/xeno
+#define CLIMATE_NONE null
+
+#define LOOT_TYPE_BEACH 1<<0
```

Comment:
Idk about operator priority for `<<` but I would wrap these in parenthesis just in case.